widgets/common/src/deploymentsView/header/FilterModal.tsx

```diff
@@ -56,19 +56,23 @@ const FilterModal: FunctionComponent<FilterModalProps> = ({
         Button,
         CancelButton,
         Dimmer,
+        Dropdown,
         Icon,
+        Input,
         Modal,
         UnsafelyTypedForm,
         UnsafelyTypedFormField
     } = Stage.Basic;
     // @ts-expect-error DynamicDropdown is not converted to TS yet
     const { DynamicDropdown } = Stage.Common;
-    const { useBoolean, useErrors } = Stage.Hooks;
+    const { useBoolean, useErrors, useInput } = Stage.Hooks;
 
     const { errors, setErrors, clearErrors, setMessageAsError } = useErrors();
     const [rulesErrorsPresent, setRuleErrorsPresent] = useState<boolean>();
 
     const [filterSaving, setFilterSaving, unsetFilterSaving] = useBoolean();
+    const [saveAsActive, activateSaveAs, deactivateSaveAs] = useBoolean();
```

Comment:
Minor:

```suggestion
    const [saveAsModeActive, activateSaveAsMode, deactivateSaveAsMode] = useBoolean();
```

The same applies to other variable/function names, to generally rename `saveAs` to `saveAsMode`

Reply:
Applied